vep/src/main/scala/VepStage.scala

```diff
@@ -45,7 +45,8 @@ class VepStage(implicit context: Context) extends Stage {
       new BootstrapScript(clusterBootstrap),
       new BootstrapScript(installScript)
     ),
-    stepConcurrency = 3
+    stepConcurrency = 3,
+    releaseLabel = ReleaseLabel("emr-6.7.0") // Need emr 6.1+ to read zstd files
```

Comment:
Don't think this is actually necessary, but I don't think it hurts either?

Reply:
(1) It won't hurt, and (2) in general I would like to slowly migrate things over so this is a good idea. It'll end up being a good marker for indicating roughly which bits have been switched to compression.